pagopa/pn-frontend
# fix(PN-11250): redirect to selfcare after saving aar in localStorage

## Short description
Redirected to selfcare after saving aar and session token in storage when user looks for login.

## List of changes proposed in this pull request
- Check if inactivityTimer before initialize inactivity listener
- Save aar token in localStorage in RouteGuard for PF and PG
- Refactor SessionGuard for PF and PG, adding condition if user is anonimous

## How to test
Login as PF through [https://cittadini.dev.notifichedigitali.it/?aar=WktSVC1EWVJOLU5WUkstMjAyNDA2LVktMV9QRi1kZmEyZDdmNC03YzFlLTRmZWYtYmUzMC02MGVhZDc4NmU3MGZfMWYxZGVkYmUtN2MzZC00YTAwLTg5OGEtNzc4MzAyM2M4YjM5](https://cittadini.dev.notifichedigitali.it/?aar=WktSVC1EWVJOLU5WUkstMjAyNDA2LVktMV9QRi1kZmEyZDdmNC03YzFlLTRmZWYtYmUzMC02MGVhZDc4NmU3MGZfMWYxZGVkYmUtN2MzZC00YTAwLTg5OGEtNzc4MzAyM2M4YjM5); check after login you are redirect to notification;

Login as PG through [https://imprese.dev.notifichedigitali.it/?aar=V1lHUC1XWVJBLU5VTUUtMjAyNDA1LVEtMV9QRy04ZmFkMGQwNy1iMTlhLTQ3YmMtOWQ2Ni1jNjllOWIxODIwZWRfYjgzNjI2MGUtYzU0NC00YTJmLTkyYzQtNWM5N2M4ZWI4MzZj](https://imprese.dev.notifichedigitali.it/?aar=V1lHUC1XWVJBLU5VTUUtMjAyNDA1LVEtMV9QRy04ZmFkMGQwNy1iMTlhLTQ3YmMtOWQ2Ni1jNjllOWIxODIwZWRfYjgzNjI2MGUtYzU0NC00YTJmLTkyYzQtNWM5N2M4ZWI4MzZj); check after login you are redirect to notification;
